### PR TITLE
Update integrated gradients example for Keras 3.

### PR DESCRIPTION
**Pull request by hertschuh**

### CODE REVIEW

File: examples/vision/integrated_gradients.py [hunk 1]
```diff
@@ -55,12 +55,15 @@
...
 
 import tensorflow as tf
-from tensorflow import keras
-from tensorflow.keras import layers
-from tensorflow.keras.applications import xception
+import keras
+from keras import layers
+from keras.applications import xception
+
+keras.config.disable_traceback_filtering()
```
Comment (fchollet) [inline examples/vision/integrated_gradients.py:65]: Likewise, you can remove this line in all 3 files

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: examples/vision/integrated_gradients.py (hunk 1 above)
Comment (hertschuh) [reply examples/vision/integrated_gradients.py]: Sorry, fixed.

**Review by fchollet — APPROVED** [review]
LGTM, thank you
